tests/tests/eth_provider.rs

```diff
@@ -291,6 +291,30 @@ async fn filter_logs(filter: Filter, provider: Arc<dyn EthereumProvider>) -> Vec
     }
 }
 
+#[rstest]
+#[awt]
+#[tokio::test(flavor = "multi_thread")]
+async fn test_get_logs_limit(#[future] katana: Katana, _setup: ()) {
+    // Get the Ethereum provider from Katana.
+    let provider = katana.eth_provider();
+
+    // Add mock logs to the Katana instance's database.
+    // The number of logs added is LIMIT_LOGS + 20, ensuring there are more logs than the limit.
+    katana.add_mock_logs(((*LIMIT_LOGS).unwrap() + 20) as usize).await;
```

Comment:
don't you need to set this value in your test with `std::env::set_var`?

Reply:
Yes indeed, surprisingly it still worked... I just set it.